cdci_data_analysis/plugins/dummy_plugin/instr_catalog.py

```diff
@@ -66,9 +66,9 @@ def __init__(self,
 
 
         self.add_column(data=NEW_SOURCE, name='NEW_SOURCE')
-        self.add_column(data=ISGRI_FLAG, name='ISGRI_FLAG', dtype=np.int)
-        self.add_column(data=FLAG, name='FLAG', dtype=np.int)
-        self.add_column(data=ERR_RAD, name='ERR_RAD', dtype=np.float)
+        self.add_column(data=ISGRI_FLAG, name='ISGRI_FLAG', dtype=np.int32)
+        self.add_column(data=FLAG, name='FLAG', dtype=np.int32)
+        self.add_column(data=ERR_RAD, name='ERR_RAD', dtype=np.float64)
```

Comment:
how do you choose the precision (32 or 64)? It's in the table?

Reply:
I've chosen it rather arbitrarily, it's just a dummy plugin for tests. I decided that int32 is enough for flags.
Or we could use python types here, this corresponds to what was previously meant by deprecated types, as far as I understand from the deprecation notice.

Reply:
> I've chosen it rather arbitrarily, it's just a dummy plugin for tests.

right, indeed

> I decided that int32 is enough for flags. 

> Or we could use python types here, this corresponds to what was previously meant by deprecated types, as far as I understand from the deprecation notice.

This seems to be the most straightforward way, if it works